### PR TITLE
chore: release 3.0.0-alpha.11

### PR DESCRIPTION
**Pull request by jooola**
Preparing the 3.0.0-alpha.11 release.

What do we want to merge before releasing this ?
- #1568 (only website related but this new release note will land there)
- #1608
-  #1625
- #1624 ? I think with a bit of testing before releasing this change is not that risky.
- #1606
- #1663

### CODE REVIEW

**Comment by jooola** [comment]
Cool, I guess all my wishes are answered, I'll start doing some heavy testing.

**Comment by paddatrapper** [comment]
I'll be able to help with that next week

**Comment by jooola** [comment]
Do you have any particular issue you want to be fixed before this new version is released ?

**Comment by paddatrapper** [comment]
Potentially #1606, but if that is an upstream issue we should just package liquidsoap 1.4.4 for Focal and use the PPA

**Comment by jooola** [comment]
@paddatrapper I think we are ready to release. The past week allowed us to merge a lot of fixes for this release to be ""almost perfect"".

I have it running in production already (bullseye) and I am happy with it.

**Comment by jooola** [comment]
I just tested the upgrade procedure on vagrant buster from alpha.10 to the current main.

I added some fixes in the upgrade document, and added some notables changes that were missing.

**Comment by paddatrapper** [comment]
Need addressing before we can release:

- #1684
- #1683

File: website/versioned_docs/version-3.0.0-alpha.11/admin-manual/setup/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Follow [the install guide](./install.md#download) to download and install the new version, and re-run the `./install` script with the same arguments you used during the initial install.
+
+## Apply upgrade instructions
+
+Be sure to carefully read **all** the [releases notes](../../../releases), from your current version to the targeted version, to apply upgrade or breaking changes instructions to your installation.
+
+## Restart the services
+
+Restart all the services to make sure all the changes are applied.
```
[the remark this answers is not in the document]
Comment (jooola) [reply website/versioned_docs/version-3.0.0-alpha.11/admin-manual/setup/upgrade.md:28]: Note: I'll remove the versioned docs for now, we will be able to freeze a version once we introduce breaking changes for alpha.12/beta.1

**Comment by paddatrapper** [comment]
Release to do:

- [ ] Merge #1697
- [ ] Test Ubuntu 18.04 and ensure playout working (#1606)
- [ ] Fix #1684
- [ ] Update release notes
- [ ] Test, test, test

**Comment by jooola** [comment]
We also need to move and close https://github.com/libretime/LibreTime.github.io/issues/1 to this repo

**Comment by paddatrapper** [comment]
Shall I move https://github.com/libretime/LibreTime.github.io/issues/1 here after the release?

**Comment by jooola** [comment]
> Shall I move https://github.com/libretime/LibreTime.github.io/issues/1 here after the release?

Yup, that would be the best I think.

**Comment by jooola** [comment]
@paddatrapper I wasn't able to move the issue back to the main repository, could you take a look ?

**Comment by paddatrapper** [comment]
Nor was I...

**Pull request merged**